src/main/java/org/Roclh/handlers/commands/manager/ExportCsvCommand.java

```diff
@@ -0,0 +1,109 @@
+package org.Roclh.handlers.commands.manager;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.Roclh.data.Role;
+import org.Roclh.data.entities.TelegramUserModel;
+import org.Roclh.data.entities.UserModel;
+import org.Roclh.data.services.TelegramUserService;
+import org.Roclh.data.services.UserService;
+import org.Roclh.handlers.commands.AbstractCommand;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.methods.send.SendMediaGroup;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.media.InputMedia;
+import org.telegram.telegrambots.meta.api.objects.media.InputMediaDocument;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@Slf4j
+public class ExportCsvCommand extends AbstractCommand<SendMediaGroup> {
+
+
+    private final UserService userManager;
+
+    public ExportCsvCommand(TelegramUserService telegramUserService, UserService userManager) {
+        super(telegramUserService);
+        this.userManager = userManager;
+    }
+
+    @Override
+    public boolean isManager(Long userId) {
+        return telegramUserService.isAllowed(userId, Role.ROOT);
+    }
+
+    @Override
+    public SendMediaGroup handle(Update update) {
```

Comment:
Maybe add argument to command to allow select database to export 

e.g. 
exportscv users    -> only users.csv appears
exportcsv tgusers -> only tgusers.csv returned

leave ability to request all csv files with exportcsv without param